wp-content/plugins/jetpack/_inc/build/contact-form/js/editor-view.min.asset.php

```diff
@@ -0,0 +1 @@
+<?php return array('dependencies' => array(), 'version' => 'dc2473cb375a85ab1af41c0c38268c5c');
```

Comment:
:no_entry_sign: **Error( severity 5 )**: When a multi-item array uses associative keys, each value should start on a new line (*WordPress.Arrays.ArrayDeclarationSpacing.AssociativeArrayFound*).